docs/source/intro/tutorial_page.rst

```diff
@@ -0,0 +1,38 @@
+.. _intro:tutorial_page:
+
+**********
+Tutorials
+**********
+
+Welcome to the AiiDA tutorial!
+
+AiiDA has an extensive and well spread tutorials, it's the best to first start the :ref:`Basic tutorial <tutorial:basic>`
+
+
+.. toctree::
+   :maxdepth: 1
+
+   tutorial
+
+
+
+.. grid:: 1 1 2 2
+   :gutter: 3
+
+   .. grid-item-card:: :fa:`baby;mr-1` Basic tutorial
+
+      .. button-ref:: tutorial:basic
+         :ref-type: ref
+         :click-parent:
+         :class: btn-link
+
+         First time users: Get your feet wet with AiiDA!
+
+      Learn how to decorate your Python function quickly, and take advantage of features like automatic data storage, job submission, background process monitoring, and provenance graph visualisation.
+
+
+   .. grid-item-card::  :fa:`fire;mr-1` Archive tutorials
+
+      `Dive deeper into AiiDA! <https://aiida-tutorials.readthedocs.io/en/latest/>`_
+
+      You can get access to detailed tutorials designed for hands-on workshops. There you will find YouTube links to relevant  presentaions as well as a list of all the hands-on materials from past tutorials that the AiiDA team has organised since 2016.
```

Comment:
```suggestion
      `Additional tutorials <https://aiida-tutorials.readthedocs.io/en/latest/>`_

      These tutorials were part of in-person hands-on workshops that have been organized since 2016.
      In addition to tutorials, they include Youtube videos of recorded presentations and demonstrations by the AiiDA team.
```